exporters/jaeger-thrift/src/main/java/io/opentelemetry/exporter/jaeger/thrift/JaegerThriftSpanExporterBuilder.java

```diff
@@ -46,6 +47,7 @@ public JaegerThriftSpanExporterBuilder setEndpoint(String endpoint) {
    * @return a new exporter's instance.
    */
   public JaegerThriftSpanExporter build() {
+    ThriftSender thriftSender = this.thriftSender;
```

Comment:
These fix the issue that `build` was mutating the builder which shouldn't happen.